.github/workflows/dapr-bot.yml

```diff
@@ -106,33 +106,6 @@ jobs:
                   
                   console.log(`Trigger E2E test for ${JSON.stringify(testPayload)}`);
                 }
-              } else if (commentBody.indexOf("/ok-to-wintest") == 0) {
```

Comment:
I have one suggestion; I would not ask to change it in this pr now :) it would be fair to consider these options in the future. 

Would it be good to keep per-platform test command ? in some cases, only windows test would fail.

for example.

* ok-to-test + scheduled test -> linux + windows
* ok-to-linuxtest -> only linux
* ok-to-wintest -> only windows

In terms of implementation, we can use single workflow file, but we can quit non-target test earlier.

Reply:
Ok, why don't I create a followup issue for this? Do you really think there is utility in running them separately?

Reply:
https://github.com/dapr/dapr/issues/1998